internal/spokes/spokes_test.go

```diff
@@ -28,7 +28,7 @@ func TestCheckHiddenRefs(t *testing.T) {
 		{"refs/remotes/origin/main", hiddenRefs, false},
 		{"refs/__gh__/pull/99986/rebase", hiddenRefs, true},
 		{"refs/gh/merge_queue/156066/6e33e3a2c52017bec941ffd6f15c20a1ae002ad9", hiddenRefs, true},
-		{"refs/pull/95628/head", hiddenRefs, true},
```

Comment:
Is this no longer useful?  If not we can probably remove the `refs/pull` pattern from `hiddenRefs`, as I don't see it used anywhere anymore.

Reply:
Nah, I don't know why I removed it here, sorry, my bad.

Added it back in 8f1d8b6dcca4000c30d9cb4d513d507a00d34262